### PR TITLE
manual: develop/firmware: add note to fedora tab about path

### PR DESCRIPTION
**Pull request by purdeaandrei**
For reference:
Fedora changelog:
image
Fedora readme for SDCC:
image

### CODE REVIEW

**Comment by purdeaandrei** [comment]
hmm, I hope the nested tap/note/code works correctly

**Comment by purdeaandrei** [comment]
When switching to the fedora tab:

image

**Comment by purdeaandrei** [comment]
image

File: docs/manual/src/develop/firmware.rst [hunk 1]
```diff
@@ -75,6 +75,14 @@ The firmware can only be built on a Unix-like system; to develop the firmware on
 
         $ sudo dnf install -y make sdcc git python3-libusb1 libusb1
 
+    .. note ::
+
+        On Fedora the sdcc package installs its binaries into ``/usr/libexec/sdcc``, and it installs shellscripts into ``/usr/bin`` with the ``sdcc-`` prefix, that pass execution onto the real binaries in libexec. The firmware build system relies on sdcc binaries having the non-prefixed names, so it's recommended to append ``/usr/libexec/sdcc`` to the ``PATH`` environment variable. If you use bash, then you can append the following line to the end of your ``.bashrc``, and then open a new terminal:
```
Comment (whitequark) [inline docs/manual/src/develop/firmware.rst:80]: This is a lot of text that's not really necessary. The additional explanation adds very little, and anyone building the firmware themselves is going to know how to manipulate `PATH`.

```suggestion
        On Fedora, the `sdcc` package installs the tools into a different location, which must be added to the search path:
```

File: docs/manual/src/develop/firmware.rst [hunk 2]
```diff
@@ -75,6 +75,14 @@ The firmware can only be built on a Unix-like system; to develop the firmware on
...
+
+        .. code:: bash
```
Comment (whitequark) [inline docs/manual/src/develop/firmware.rst:82]:
```suggestion
        .. code:: console
```

**Review by whitequark — CHANGES_REQUESTED** [review]

**Comment by purdeaandrei** [comment]
Added your suggestions, and squashed.

It now looks like this: 
image

**Comment by whitequark** [comment]
Thanks!

**Review by whitequark — APPROVED** [review]

**Pull request merged**